cpp/CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ option(BUILD_WITH_qpOASES "Compile the wrapper for qpOASES" OFF)
 option(BUILD_WITH_GLPK "Compile the wrapper for GLPK (GPL license)" OFF)
 # General options
 option(TOPPRA_DEBUG_ON "Set logging mode to on." OFF)
+option(PYTHON_BINDINGS "Build bindings for Python" ON)
```

Comment:
@jmirabel You will need to turn this off if there is no pybind installation.

Reply:
Ok. Thanks for the notice.